install.sh

```diff
@@ -247,7 +253,7 @@ do_install() {
         do_install_tar "${INSTALL_RKE2_CHANNEL}"
         ;;
     *)
-        warn "unknown installation method, '${INSTALL_RKE2_METHOD}', assuming you meant 'tar'"
+        info "unknown installation method, '${INSTALL_RKE2_METHOD}', assuming you meant 'tar'"
```

Comment:
based on feedback in slack, go ahead and delete this logging line

Reply:
Done